### DISCUSSION TITLE
开源：一场无限的游戏 - 「开源之道」

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#43] [Announcements]
# 开源：一场无限的游戏 - 「开源之道」

https://opensourceway.community/posts/opensource_culture/open-source-an-infinite-game/

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

私人偏见哈, 有限无限正好相反:

- 商业的贪婪是无限的, 原先垄断到最后狠到能将自己也吞掉...
- FLOSS/自由, 开源项目中, 个人的兴趣/精力/想象力是有限的, 正是因为有限, 大家才能真正逐步现实, 然后, 再展开下一个有限...

参考: [截拳道之道（Tao of Jeet Kune Do） (豆瓣)](https://book.douban.com/subject/25927566/)
以有间入无间,
可以无限继续的事儿, 
可能就是因为, 这事儿可以随时停止, 休息一下, 重新开始...

对不对?
